pkg/client/unversioned/remotecommand/remotecommand.go

```diff
@@ -155,18 +156,16 @@ func (e *Streamer) doStream() error {
 	}
 	defer conn.Close()
 
-	doneChan := make(chan struct{}, 2)
 	errorChan := make(chan error)
+	var wg sync.WaitGroup
 
 	cp := func(s string, dst io.Writer, src io.Reader) {
+		defer wg.Done()
```

Comment:
I just knew that not waiting for stdin to finish reading is definitely wrong, so it seemed to me to better try and trigger the bug than to leave in code that I knew was wrong. And I just failed at reproducing the bug, so I left that in :)